e2e/tests/playwright-tests/app.cks.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { test, expect } from '@playwright/test';
+import { selectors } from "./helpers/selectors";
+import { userAuthData } from './config';
+import { encryptDecryptProcedure } from "./helpers/operations";
+import * as fs from "fs/promises";
+
+test.describe.configure({ mode: 'serial' });
+
+let fileName;
+
+test.describe('For CKS user', () => {
+	test('Encrypt/decrypt data A with a CKS user X, a CKS user using the JS SDK', async ({ browser }) => {
+		const contextZ = await browser.newContext({ storageState: 'e2e/tests/playwright-tests/.auth-cks/user.json' });
+		const pageZ = await contextZ.newPage();
+		await pageZ.goto('/');
+
+		await encryptDecryptProcedure({
+			page: pageZ,
+			recipientEmail: userAuthData.secondUser.login,
+			full: true,
+		});
+
+		const downloadPromise = pageZ.waitForEvent('download');
+		await pageZ.getByText(selectors.downloadTDFFileBtnText).click();
+		const download = await downloadPromise;
+		fileName = download.suggestedFilename();
+		await download.saveAs(fileName);
+		const fileContent = await fs.readFile(fileName);
+		expect(fileContent.toString()).toBeTruthy();
+		await contextZ.close();
```

Comment:
Do you need to delete the file here? You do it below. If the file is small, it is probably fine to leave it.

```suggestion
    await fs.rm(fileName);
		await contextZ.close();
```

Reply:
I prefer to delete to avoid filename collisions,  cause during tests we create multiple file